site/src/routes/embedding-knn/AgreementClusterStaticPlot.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let data: any[] = [];
+
+  var names = data.map((d) => d["ego_primary_label"]);
+  names = names.slice(0, 20);
+  var selected: string = "normoc";
+</script>
+
+<div class="selection">
+  <b>Species:</b>
+  {#each names as name}
+    <label>
+      <input type="radio" bind:group={selected} name="names" value={name} />
+      {name}
+    </label>
+  {/each}
+</div>
+
+<h5>Distances</h5>
+<img
+  src="https://storage.googleapis.com/birdclef-2023/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-static/v1/{selected}/distances.png"
```

Comment:
Do you mind replacing these next few lines with the api endpoint instead? It'll look basically the same, but the benefit is that we can actually serve data from local disk if we wanted to.

```suggestion
  src="/api/v1/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-static/v1/{selected}/distances.png"
```